src/components/Auth/Profile/tabs/Dashboard.jsx

```diff
@@ -1,51 +1,88 @@
+import { useEffect, useState } from "react";
+import { UserApi } from "../../../../api/auth/user";
+
 export default function Dashboard() {
+  const [user, setUser] = useState(null);
+
+  useEffect(() => {
+    const fetchUser = async (id) => {
+      try {
+        const { data: userDetails } = await UserApi(id);
+        setUser(userDetails);
+      } catch (error) {
+        console.error("Error fetching user data:", error);
+      }
+    };
+
+    const token = localStorage.getItem("token");
+
+    if (!token) {
+      console.log("No token found");
+      return;
+    }
+
+    try {
+      const payload = token.split(".")[1];
+      const decodedPayload = atob(payload);
+      const decodedPayloadObject = JSON.parse(decodedPayload);
+
+      fetchUser(decodedPayloadObject?.id);
+    } catch (error) {
+      console.error("Error decoding token:", error);
```

Comment:
Do not print the error 
```suggestion
        enqueueSnackbar( "Something went wrong, Please try again later!", { variant : "error" } );
```